assets/javascripts/src/components/InputField.jsx

```diff
@@ -4,13 +4,14 @@ export class AmountInput extends React.Component {
     render() {
         const { inputAmount, symbol, refFn, ...props } = this.props;
 
-        return <span className="contribute-controls__input contribute-controls__input--amount input-text">
+        return <span className={(this.props.small ? 'contribute-controls__input-small':'contribute-controls__input') + ' contribute-controls__input--amount input-text'}>
```

Comment:
Aha, I thought the width would compete with the flex-basis, but apparently not, thanks.